packages/data-models/flowTypes.ts

```diff
@@ -362,7 +362,26 @@ export namespace FlowTypes {
     [key: string]: any;
   };
 
-  type IDynamicPrefix = IAppConfig["DYNAMIC_PREFIXES"][number];
+  const DYNAMIC_PREFIXES_COMPILER = ["gen", "row"] as const;
```

Comment:
Should "default" be here too?

Reply:
You're right, updated with [569234b](https://github.com/IDEMSInternational/open-app-builder/pull/2494/commits/569234b99d0a19553250a2f32aadde77d7328ef2)